p2p/client.go

```diff
@@ -133,6 +163,14 @@ func (c *Client) StartWithHost(ctx context.Context, h host.Host) error {
 		return err
 	}
 
+	if c.conf.BlockSyncEnabled {
+		c.logger.Debug("Setting up block sync protocol")
+		err = c.startBlockSync(ctx)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
maybe worth adding a  log here indicating that working is disabled

Reply:
added